examples/carousel/README.md

```diff
@@ -0,0 +1,38 @@
+# carousel
+
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+
+## Getting Started
+
+First, run the development server:
+
+```bash
+npm run dev
```

Comment:
minor: no need to change now, but might be best if we just support one (with the understanding that others work)